x-pack/plugins/profiling/e2e/cypress/e2e/profiling_views/differential_functions.cy.ts

```diff
@@ -0,0 +1,160 @@
+        { id: 'annualizedCo2', value: '275.62 lbs / 125.02 kg' },
+        { id: 'annualizedCost', value: '$318.32' },
+        { id: 'totalNumberOfSamples', value: '513' },
+      ].forEach((item) => {
+        cy.get(`[data-test-subj="${item.id}_value"]`).contains(item.value);
+        cy.get(`[data-test-subj="${item.id}_comparison_value"]`).should('not.exist');
+      });
+    });
+
+    it('shows empty baseline values when data is not available', () => {
+      cy.intercept('GET', '/internal/profiling/topn/functions?*').as('getTopNFunctions');
+      cy.visitKibana('/app/profiling/functions/differential', {
+        comparisonRangeFrom: rangeFrom,
+        comparisonRangeTo: rangeTo,
+      });
+      cy.wait('@getTopNFunctions');
+      cy.wait('@getTopNFunctions');
+      [
+        { id: 'overallPerformance', value: '0%' },
+        { id: 'annualizedCo2', value: '0 lbs / 0 kg', comparisonValue: '275.62 lbs / 125.02 kg' },
+        { id: 'annualizedCost', value: '$0', comparisonValue: '$318.32' },
+        { id: 'totalNumberOfSamples', value: '0', comparisonValue: '15,390' },
+      ].forEach((item) => {
+        cy.get(`[data-test-subj="${item.id}_value"]`).contains(item.value);
+        if (item.comparisonValue) {
+          cy.get(`[data-test-subj="${item.id}_comparison_value"]`).contains(item.comparisonValue);
+        }
+      });
+    });
+
+    it('show gained performance when comparison data has less samples than baseline', () => {
+      cy.intercept('GET', '/internal/profiling/topn/functions?*').as('getTopNFunctions');
+      cy.visitKibana('/app/profiling/functions/differential', {
+        rangeFrom,
+        rangeTo,
+        comparisonRangeFrom,
+        comparisonRangeTo,
+      });
+      // wait for both apis to finisto move on
+      cy.wait('@getTopNFunctions');
+      cy.wait('@getTopNFunctions');
+      [
+        { id: 'overallPerformance', value: '65.89%', icon: 'sortUp_success' },
+        {
+          id: 'annualizedCo2',
+          value: '275.62 lbs / 125.02 kg',
+          comparisonValue: '94.02 lbs / 42.65 kg (65.89%)',
+          icon: 'comparison_sortUp_success',
+        },
+        {
+          id: 'annualizedCost',
+          value: '$318.32',
+          comparisonValue: '$108.59 (65.89%)',
+          icon: 'comparison_sortUp_success',
+        },
+        {
+          id: 'totalNumberOfSamples',
+          value: '513',
+          comparisonValue: '175 (65.89%)',
+          icon: 'comparison_sortUp_success',
+        },
+      ].forEach((item) => {
+        cy.get(`[data-test-subj="${item.id}_value"]`).contains(item.value);
+        cy.get(`[data-test-subj="${item.id}_${item.icon}"]`).should('exist');
+        if (item.comparisonValue) {
+          cy.get(`[data-test-subj="${item.id}_comparison_value"]`).contains(item.comparisonValue);
+        }
+      });
+    });
+    // Flaky test, skipping it for now
+    it('show lost performance when comparison data has more samples than baseline', () => {
```

Comment:
```suggestion
    // Flaky test, skipping it for now
    it.skip('show lost performance when comparison data has more samples than baseline', () => {
```

Since the comment mentions that this test is flaky, do you want `it.skip` instead?

Reply:
I actually forgot to remove the comment. I fixed the test.